src/test/java/org/alfresco/repo/search/impl/querymodel/impl/db/DBQueryTest.java

```diff
@@ -66,11 +66,7 @@
 import org.alfresco.service.cmr.repository.StoreRef;
 import org.alfresco.service.cmr.repository.datatype.DefaultTypeConverter;
 import org.alfresco.service.cmr.repository.datatype.Duration;
-import org.alfresco.service.cmr.search.QueryConsistency;
-import org.alfresco.service.cmr.search.ResultSet;
-import org.alfresco.service.cmr.search.ResultSetRow;
-import org.alfresco.service.cmr.search.SearchParameters;
-import org.alfresco.service.cmr.search.SearchService;
+import org.alfresco.service.cmr.search.*;

```

Comment:
We don't do wildcard import. Can we please fix this?

Reply:
Removed wildcard.